tests/main.test.ts

```diff
@@ -1,34 +1,19 @@
 import { RestEndpointMethodTypes } from "@octokit/plugin-rest-endpoint-methods";
-/* eslint-disable @typescript-eslint/naming-convention */
-// @ts-expect-error package name is correct, TypeScript doesn't recognize it
 import { afterAll, afterEach, beforeAll, describe, expect, it, jest, mock, spyOn } from "bun:test";
 import { config } from "dotenv";
 import { GitHubContext } from "../src/github/github-context";
 import { GitHubEventHandler } from "../src/github/github-event-handler";
 import { getConfig } from "../src/github/utils/config";
 import worker from "../src/worker";
 import { server } from "./__mocks__/node";
+import { WebhooksMocked } from "./__mocks__/webhooks";
 
-mock.module("@octokit/webhooks", () => ({
+void mock.module("@octokit/webhooks", () => ({
```

Comment:
what's the purpose of `void`?

Reply:
Return of it is `void | Promise<void>` so it just avoids TS complaining about not awaiting a promise. It's a module so it was safe to disable. Eslint is the one to complain
```
✖ eslint --fix:
/Users/mentlegen/WebstormProjects/ubiquibot-kernel/tests/main.test.ts
  11:1  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
✖ 1 problem (1 error, 0 warnings)
```

Reply:
What about the `.catch`?

Reply:
@0x4007 If the mock truly fails, I'd rather have the test suite failing than the error being caught and the tests running with an un-mocked module.